app/models/post_action_type.rb

```diff
@@ -14,6 +14,11 @@ def expire_cache
   class << self
     attr_reader :flag_settings
 
+    def clear_cache!
```

Comment:
I'm curious where this is called from and I'm wondering how it will work in a multiple process environment where we need to clear the cache from all Unicorn processes that are running at the same time.

Reply:
Yes sorry I haven’t thought this part through yet, I added it hastily. You’re right we need something better

Reply:
Maybe we can rely on Redis instead to cache the necessary data? I'm looking at the various spot `all_flags` is called and it seems like we only use data from the columns. Therefore, we can actually just cache the data as a hash in Redis instead of creating ActiveRecord objects and memoizing them. 

Having the data in Redis will also make it easier to clear the cache.

Reply:
OK cool will try that